pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/v3/AdminApiTransactionTest.java

```diff
@@ -373,6 +375,23 @@ public void testGetPendingAckInternalStats() throws Exception {
         TransactionImpl transaction = (TransactionImpl) getTransaction();
         final String topic = "persistent://public/default/testGetPendingAckInternalStats";
         final String subName = "test";
+        try {
+            admin.transactions()
+                    .getPendingAckInternalStatsAsync(topic, subName, true).get();
```

Comment:
Maybe we need to add ``Assert.fail()`` here.

Reply:
yes, fixed